lib/config.js

```diff
@@ -38,6 +38,11 @@ var Config = module.exports = function (cfg) {
   this.scripts = this._loadScripts(this.archetypes);
 };
 
+// Expose `require()` for testing.
```

Comment:
I've got a more full explanation here -- https://github.com/FormidableLabs/builder/pull/85/files#diff-4ffc5fe4c4f93897ccc6fb2331ae5e44R56

Basically (1) `fs` mocking doesn't work with Node 4+, and (2) you want to defeat the require cache.

But this code is mostly here to just _allow_ the tests to decide how to override / stub...